src/test/scala/uk/gov/hmrc/play/views/helpers/InputRadioGroupSpec.scala

```diff
@@ -31,7 +30,7 @@ class InputRadioGroupSpec extends WordSpec with Matchers with MessagesSupport {
 
   val max = 10
 
-  def dummyForm =
+  def dummyForm: Form[DummyFormData] =
```

Comment:
We can probably remove this explicit type here, I think we just added when we were refactoring.

Reply:
good catch!

- [ ] remove explicit type that got left in